hw1-app/src/components/SearchBar.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+let SearchBar = (props) => 
+    <div className="wrap">
```

Comment:
We are creating React functional components always as const as it can be 'changed'.